### ISSUE TITLE
Trash icon/folder not visible and Joplin desktop app freezes completely when scrolling to bottom of sidebar

### ISSUE THREAD

**Issue by luigicarrini** [andrejilderda/joplin-macos-native-theme#144]
### Operating system

macOS
Joplin version

3.0.13
Desktop version info

Joplin for Desktop

Copyright © 2016-2024 Laurent Cozic
Joplin 3.0.13 (prod, darwin)

Client ID: 55216ab8a16342a2ac3a307fad5b644b
Sync Version: 3
Profile Version: 47
Keychain Supported: Yes

Revision: https://github.com/laurent22/joplin/commit/599cf5b86f379c80ba2a2346a43607b53e45eca7

Backup: 1.4.1
Favorites: 1.3.2
macOS theme: 1.5.7
Note list (Preview): 1.1.0
Note list and sidebar toggle buttons: 1.0.3
Note overview: 1.7.1

### Current behaviour

1. Start Joplin
2. Scroll to bottom of sidebar
3. No trash icon/folder visible in sidebar
4. Joplin desktop app freezes, remains completely unresponsive
5. Closing the Joplin app is only possible via Force Quit in MacOS

### Expected behaviour

Trash icon/folder visible.
Joplin app doesn't freeze/crash.

#### Logs

[Intervention] Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: file:///Applications/Joplin.app/Contents/Resources/app.asar/vendor/lib/roboto-fontface/fonts/roboto/Roboto-Bold.woff2

**Comment by hohowt** [comment]
same question in mac/arch

**Comment by zhouxinghong** [comment]
Unable to adapt jopin 3.0.14

Windows 10 19044
Desktop version 3.0.13

**Comment by hololee** [comment]
same

**Comment by andrejilderda** [comment]
Do you have emoji's for folders selected? In that case, does [this workaround](https://github.com/andrejilderda/joplin-macos-native-theme/issues/96#issuecomment-1553517708) work? Unfortunately I can't reproduce this.

**Comment by luigicarrini** [comment]
Yes, thanks. The workaround of replacing joplin folder emojis with emojis in folder titles solved this issue for me.

**Comment by andrejilderda** [comment]
Duplicate of #96 .
